easybuild/easyconfigs/d/deepTools/deepTools-3.3.1-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'Bundle'
+
+name = 'deepTools'
+version = '3.3.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://deeptools.readthedocs.org/'
+description = """deepTools is a suite of python tools particularly developed for the efficient analysis of
+ high-throughput sequencing data, such as ChIP-seq, RNA-seq or MNase-seq."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('Pysam', '0.15.1', versionsuffix),
+    ('matplotlib', '3.0.0', versionsuffix),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
```

Comment:
both lines can be removed when using `PythonBundle`

Reply:
Done.